akka-bbb-apps/src/main/scala/org/bigbluebutton/core/models/Plugins.scala

```diff
@@ -112,6 +116,26 @@ object PluginModel {
     val baseUrl = plugin.manifest.url.substring(0, plugin.manifest.url.lastIndexOf('/') + 1)
     baseUrl + relativeUrl
   }
+  private def createFinalJavascriptEntrypointUrl(plugin: Plugin, jsEntrypointAbsoluteUrl: String): String = {
+    val maybeVersion = for {
+      manifest <- plugin.manifest.content
+      version  <- manifest.pluginVersion
+      if Version.isValid(version)
```

Comment:
The validation should be more tolerant. We don't want to force a standard for the version format.
`isValid(String version, boolean strictly)` should be used disabling the strict mode.
It will allow versions like `1.0` and `v1.2.3` that is usual and safe.